Tasks/KubernetesV1/task.json

```diff
@@ -152,21 +152,49 @@
         {
             "name": "useConfigurationFile",
             "type": "boolean",
-            "label": "Use configuration files",
+            "label": "Use configuration",
             "defaultValue": "false",
-            "helpMarkDown": "Use Kubernetes configuration file with the kubectl command. Filename, directory, or URL to Kubernetes configuration files can also be provided.",
+            "helpMarkDown": "Use Kubernetes configuration with the kubectl command. An inline script, filename, directory, or URL to Kubernetes configuration files can be provided.",
```

Comment:
Although the documentation says it supports URL, it doesn't right now. 
Perhaps a different PR to handle this issue.